tests/test_contracts.py

```diff
@@ -1,7 +1,12 @@
-from unittest import TextTestResult
+import sys
+from unittest import TextTestRunner
 
+from twisted.internet.defer import TimeoutError, CancelledError
+from twisted.python.failure import Failure
 from twisted.trial import unittest
+from scrapy.commands.check import TextTestResult
```

Comment:
could you please move this import after a blank line? Imports are grouped in 3 groups: stdlib imports, third-party imports, Scrapy imports.